redesign/cs-emf.html

```diff
@@ -55,11 +55,11 @@ <h1 class="dwyl-dark-gray f3-plus w-90 tc center">Online Membership Platform</h1
       <article class="w-50-ns dtc v-top istok-web lh-copy dwyl-dark-gray">
         <h3 class="b mb2">Project: <span class="normal">CE100</span></h3>
         <div class="mb4">
-          <a href="#" class="dib link mv2 pa2 dwyl-bg-yellow dwyl-dark-gray br2 f5 shadow-4">
+          <a href="https://www.ellenmacarthurfoundation.org/ce100" class="dib link mv2 pa2 dwyl-bg-yellow dwyl-dark-gray br2 f5 shadow-4">
```

Comment:
This is not the URL to the live app, but we also don't have permissions to share this yet, so I would comment this `<a>` out altogether for now. GH is open source so 👍

Reply:
Apologies, this is what @markwilliamfirth asked me to put in: https://github.com/dwyl/dwyl-site/issues/291#issuecomment-324310716

Reply:
@iteles can you add the correct link to the case study content doc?
https://github.com/dwyl/hq/issues/377